js/assessments/sentenceLenghtInTextAssessment.js

```diff
@@ -0,0 +1,54 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var countTooLongSentences = require( "./checkForTooLongSentences.js" );
+
+/**
+ * Calculates sentence length score
+ * @param {array} sentences The array containing sentences.
+ * @param {object} i18n The object used for translations.
+ * @returns {object} Object containing score and text.
+ */
+var calculateSentenceLengthResult = function( sentences, i18n ) {
+	var totalSentences = sentences.length;
```

Comment:
Seeing as this variable is only being used once, it might be wise to just directly call sentences.length on line 14.